### ISSUE TITLE
Pass deploy_key to bundle_install resource?

### ISSUE THREAD

**Issue by giorgian** [poise/application_ruby#78]
Issue related to [this pull request](https://github.com/poise/poise-ruby/pull/9).

I'm trying to deploy a Rails application with this cookbook, but it fails because our Gemfile includes some private repos.

It would be nice to automatically share `application_git`'s `deploy_key` attribute to `bundle_install`—I'd submit a pull request, but I have no idea how to do that.

**Comment by coderanger** [comment]
Yeah, the right way is probably to add a git_ssh into the application environment on that side. I'll see if I can find time to add it, but probably won't be immediate so I would move forward with the `ENV['GIT_SSH']` trick I mentioned on the other ticket.

**Comment by giorgian** [comment]
Sure, I'll try. Is there a way to obtain the ssh wrapper path created by application_git, so that I can set `ENV['GIT_SSH']` accordingly?
